### DISCUSSION TITLE
How to set up a httpS server?

### DISCUSSION THREAD

**Discussion by DiegoMartinezRdg** [cesanta/mongoose#1862] [Ask questions, get answers] [answered]
I'm unable to find how to set a httpS server using mongoose server. 
How is this done?

**Comment by scaprile** [comment, marked as answer]
Follow this tutorial and related links https://mongoose.ws/tutorials/tls/

**Reply by DiegoMartinezRdg** [reply]
Thank you, I missed it. 
Sorry for the inconvenience.

**Reply by scaprile** [reply]
No problem
